packages/admin/src/components/Admin/StyledComponents.js

```diff
@@ -1,39 +1,35 @@
 import styled from 'styled-components'
 import SlideMenu from 'react-burger-menu/lib/menus/slide'
-import {StyledH1} from 'tocco-ui'
+import {theme} from 'tocco-ui'
 
-export const StyledTitle = styled(StyledH1)`
- && {
-   color: white;
-   padding-left: 8px;
-   padding-top: 5px;
-   font-size: 2.5rem;
- }
-`
+import toccoLogo from '../Login/img/tocco-circle.svg'
```

Comment:
tocco-circle.svg is a shared image. i dont think it should be located in the Login folder.